examples/opengl2_example/imgui_impl_glfw_gl2.cpp

```diff
@@ -75,7 +75,14 @@ void ImGui_ImplGlfwGL2_RenderDrawData(ImDrawData* draw_data)
     GLint last_polygon_mode[2]; glGetIntegerv(GL_POLYGON_MODE, last_polygon_mode);
     GLint last_viewport[4]; glGetIntegerv(GL_VIEWPORT, last_viewport);
     GLint last_scissor_box[4]; glGetIntegerv(GL_SCISSOR_BOX, last_scissor_box); 
+#ifndef __EMSCRIPTEN__
     glPushAttrib(GL_ENABLE_BIT | GL_COLOR_BUFFER_BIT | GL_TRANSFORM_BIT);
+#else
+    // Methods gl[Push/Pop]Attrib now not implemented in emscripten's library_gl.js
+    GLboolean imgui_GL_ENABLE_BIT_was_enabled = glIsEnabled(GL_ENABLE_BIT);
+    GLboolean imgui_GL_COLOR_BUFFER_BIT_was_enabled = glIsEnabled(GL_COLOR_BUFFER_BIT);
+    GLboolean imgui_GL_TRANSFORM_BIT_was_enabled = glIsEnabled(GL_TRANSFORM_BIT);
+#endif
```

Comment:
The emscripten changes are unrelated to the title and intended purpose of the PR. They are useful are a reference because I would like to support emscripten more easily (also see https://github.com/ocornut/imgui/pull/336).

However note that your replacement code for `glPushAttrib()` is incorrect.
`glPushAttrib(GL_ENABLE_BIT)` alone with old style GL saves a _lot_ more thing.
https://www.khronos.org/registry/OpenGL-Refpages/gl2.1/xhtml/glPushAttrib.xml
I would recommend using another breed of OpenGL for the Emscripten examples, or just skipping this push/pop for now.